person_class.rb

```diff
@@ -1 +1,18 @@
+class Person
+  attr_accessor :name, :age, :parent_permission
+  attr_reader :id
 
+  def initialize(age, parent_permission: true, name: 'Unknown')
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
```

Comment:
- [ ] You should add the instance var `@id` and it can be generated randomly, it can be similar to 👉🏾  `@id = Random.rand(1..1000)`